htdocs/install/mysql/tables/llx_element_type.sql

```diff
@@ -0,0 +1,33 @@
+-- ============================================================================
+-- Copyright (C) 2008-2011	Laurent Destailleur	<eldy@users.sourceforge.net>
+-- Copyright (C) 2011		Regis Houssin		<eldy@users.sourceforge.net>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ============================================================================
+-- Table used for relations between elements of different types:
+-- invoice-propal, propal-order, etc...
+-- ============================================================================
+
+create table llx_element_type
```

Comment:
What is the goal of this table ?
The direction we must go is to follow syntax rules (with hack to manage exception that we should remove version after version), instead of adding tables for conventions purposes

Reply:
I working on a element type descriptor storable in database.
It will be usable for getElementProperties (in my TODO)
it's goal is not to create link but describe an element like getElementProperties does.

I named it llx_element_type because in Dolibarr the var I saw was named like this, but we can change this table name as llx_element_properties or an other name more appropriate...